ui/DesignSystem/Component/Illustration.svelte

```diff
@@ -546,5 +546,21 @@
       7.45457C15.2806 12.5048 19.5315 12.6427 19.5315 12.6427C19.5315 12.6427
       17.8043 11.7216 16.8524 9.76546V9.76546Z"
       fill="#292F33" />
+  {:else if variant === Variant.Computer}
```

Comment:
I wonder how I didn't realise this earlier, but all the illustrations are just emojis from the twemoji set. Which we already have integrated. So we could potentially get rid of all these custom illustration SVGs.

```diff
diff --git a/ui/Modal/ManageRemotes.svelte b/ui/Modal/ManageRemotes.svelte
index a49c5fc..b001d44 100644
--- a/ui/Modal/ManageRemotes.svelte
+++ b/ui/Modal/ManageRemotes.svelte
@@ -1,12 +1,9 @@
 <script lang="ts">
-  import { Variant as IllustrationVariant } from "../src/illustration";
-
   import {
-    Illustration,
     List,
     RemoteListItem,
   } from "../DesignSystem/Component";
-  import { Button, Input } from "../DesignSystem/Primitive";
+  import { Button, Emoji, Input } from "../DesignSystem/Primitive";

   const remotes = [
     {
@@ -56,9 +53,8 @@
 </style>

 <div data-cy="remotes-modal" class="container">
-  <Illustration
-    style="margin-bottom: 1.5rem;"
-    variant={IllustrationVariant.Computer} />
+  <Emoji emoji={'💻'} size="huge" style="margin-bottom: 1.5rem;" />
+
   <h1>Manage remotes</h1>
   <div class="input">
     <Input.Text
```

Emoji
<img width="50%" alt="Screenshot 2020-10-12 at 15 27 32" src="https://user-images.githubusercontent.com/158411/95751868-79ef6e80-0c9f-11eb-9cf7-c307c2909f1f.png">

vs

Illustration

<img width="50%" alt="Screenshot 2020-10-12 at 15 28 01" src="https://user-images.githubusercontent.com/158411/95751907-88d62100-0c9f-11eb-879b-5e2c8f4e43b4.png">

Reply:
nice

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1014/commits/903c0e71db7234671546ea6ece63c1cebdb3dee9

Reply:
Ended up using the <Illustration> component which is then in turn using the <Emoji> component. Made more sense like this.